src/views/landing.jsx

```diff
@@ -1,32 +1,63 @@
+          <div className="hidden sm:mb-8 sm:flex sm:justify-center"></div>
+          <div className="text-center">
+            <h1 className="text-4xl font-bold tracking-tight text-gray-900 sm:text-6xl">
+              facebook clone
             </h1>
-            <p className='mt-6 text-lg leading-8 text-gray-600'>
-              connections made easy - inspiration just a tap away.
+            <p className="mt-6 text-lg leading-8 text-gray-600">
+              Connect with friends and the world around you on Facebook.
             </p>
-
-            <div className='mt-10 flex items-center justify-center gap-x-6'>
-              <Link
-                to='/login'
-                className='rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600'>
-                Log in
-              </Link>
-              <Link
-                to='/signup'
-                className='rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600'>
-                Sign up
-              </Link>
-            </div>
+            {currentUser === null ? (
+              <div className="mt-10 flex items-center justify-center gap-x-6">
+                <Link
+                  to="/login"
+                  className="rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+                >
+                  Log in
+                </Link>
+                <Link
+                  to="/signup"
+                  className="rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+                >
+                  Sign up
+                </Link>
+              </div>
+            ) : (
+              <div className="mt-10 flex items-center justify-center gap-x-6">
+                <Link
+                  to="/dashboard/all-recipes"
+                  className="rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+                >
+                  dashboard
+                </Link>
+                <div className="rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
```

Comment:
this doesn't seem to be clickable element. Maybe make it a button or a link  and attach a logout function to it. maybe direct the user back to the landing page.